pkg/hubble/relay/pool/manager.go

```diff
@@ -178,7 +178,8 @@ func (m *Manager) manageConnections() {
 			m.mu.Lock()
 			p := m.peers[name]
 			m.mu.Unlock()
-			m.connect(p)
+			// a connection request has been made, make sure to attempt a connection
+			m.connect(p, true)
```

Comment:
Note to reviewers, since it's not clear from context: `m.connect()` will not reconnect if there is already a re-connection attempt in progress. So it's fine if the same peer is reported offline multiple times, we will still only do one actually re-connection attempt in `m.connect`.